### PR TITLE
Feat: TFRS: Add Redis Service for organization balance

### PR DESCRIPTION
**Pull request by prv-proton**
TFRS: Add Redis Service for Local Development
[#1078](https://github.com/bcgov/lcfs/issues/1078)

TFRS: Setup Redis Service and Call LCFS Cache for Available Balances
[#1074](https://github.com/bcgov/lcfs/issues/1074)

### CODE REVIEW

File: switch_application.sh [hunk 1]
```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```
Comment (AlexZorkin) [inline switch_application.sh:1]: Can you add some more documentation here to explain the purpose and use case for this shell script please.

File: backend/api/app.py [hunk 2]
```diff
@@ -51,11 +54,27 @@ def ready(self):
...
             except RuntimeError as error:
                 print('Startup checks failed. Not starting.')
                 print(error)
                 exit(-1)  # Django doesn't seem to do this automatically.
 
+    def seed_redis_cache(self):
+        """Seed initial data into Redis cache"""
+        try:
+            # Check if data has already been seeded to avoid duplicates
+            if not redis_cache.get('redis_seeded'):
```
Comment (AlexZorkin) [inline backend/api/app.py:68]: We have to ensure this flag is set on the LCFS side as well when it populates its live cache. Please just confirm that is the case.

**Review by AlexZorkin — CHANGES_REQUESTED** [review]
Looks good just a couple items to add. ty

File: switch_application.sh (hunk 1 above)
Comment (prv-proton) [reply switch_application.sh:1]: updated documentation. 
I was hoping to not include the redis and rabbitmq into each other docker-compose files, instead we could just switch between tfrs and lcfs using this script while ensuring redis pod from LCFS container is active while tfrs application is running and rabbitmq from TFRS container is running while LCFS application is running.

File: backend/api/app.py (hunk 2 above)
Comment (prv-proton) [reply backend/api/app.py:68]: sure, will update LCFS repo as well.

File: switch_application.sh (hunk 1 above)
Comment (AlexZorkin) [reply switch_application.sh:1]: That's great thank you.

**Review by AlexZorkin — APPROVED** [review]
Great solution to the two separate app services. ty

**Pull request merged**